src/js/components/Tab/doc.js

```diff
@@ -56,6 +56,9 @@ export const themeDoc = {
           light: 'black',
         },
       },
+      disabled: {
+        // color: undefined,
+      },
```

Comment:
We should actually stay away from nested objects on the default value, we should break the object down to separate props and that will probably take place on a separate PR. For now, can you please create a new doc entry for `tab.border.disabled.color`?

Reply:
Done.